package.json

```diff
@@ -57,52 +57,55 @@
   },
   "private": false,
   "dependencies": {
-    "@angular/animations": "~7.2.0",
-    "@angular/common": "~7.2.0",
-    "@angular/compiler": "~7.2.0",
-    "@angular/core": "~7.2.0",
-    "@angular/forms": "~7.2.0",
-    "@angular/http": "~7.2.0",
-    "@angular/platform-browser": "~7.2.0",
-    "@angular/platform-browser-dynamic": "~7.2.0",
-    "@angular/platform-server": "^7.2.0",
-    "@angular/pwa": "^0.12.0",
-    "@angular/router": "~7.2.0",
-    "@angular/service-worker": "~7.2.0",
+    "@angular/animations": "~7.2.11",
+    "@angular/common": "~7.2.11",
+    "@angular/compiler": "~7.2.11",
+    "@angular/core": "~7.2.11",
+    "@angular/forms": "~7.2.11",
+    "@angular/http": "~7.2.11",
+    "@angular/platform-browser": "~7.2.11",
+    "@angular/platform-browser-dynamic": "~7.2.11",
+    "@angular/platform-server": "^7.2.11",
+    "@angular/pwa": "^0.13.7",
+    "@angular/router": "~7.2.11",
+    "@angular/service-worker": "~7.2.11",
     "@compodoc/compodoc": "^1.1.8",
     "@ng-bootstrap/ng-bootstrap": "^4.0.1",
     "@ng-select/ng-select": "^2.13.2",
-    "@ngrx/effects": "~7.0.0",
-    "@ngrx/router-store": "~7.0.0",
-    "@ngrx/store": "~7.0.0",
+    "@ngrx/effects": "~7.4.0",
+    "@ngrx/router-store": "~7.4.0",
+    "@ngrx/store": "~7.4.0",
     "@nguniversal/express-engine": "^7.0.2",
     "@nguniversal/module-map-ngfactory-loader": "^7.0.2",
     "@types/googlemaps": "^3.30.13",
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^4.2.1",
     "core-js": "^2.5.7",
+    "express": "^4.15.2",
     "hamburgers": "^1.1.3",
     "i18next": "^15.0.6",
     "i18next-xhr-backend": "^2.0.1",
-    "ngrx-store-localstorage": "^5.1.0",
-    "rxjs": "^6.3.3",
+    "jquery": "1.9.1",
```

Comment:
Please remove `jquery`. I believe it's needed for parts of bootstrap that we don't use.

Reply:
We get warnings if we don't include it

Reply:
I believe that's fine, we had those warnings before.